sdk/core/core-amqp/src/index.node.ts

```diff
@@ -1 +0,0 @@
-export { AadTokenProvider } from "./auth/aad";
```

Comment:
There is a reference to this `index.node.ts` file somewhere and a corresponding browser one. We should remove them as well

Reply:
There is a reference to this in package.json file still

Reply:
Aah I  just removed aad from all the files and forgot to remove index.node